ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/SnapWorldDownloadState.java

```diff
@@ -111,7 +110,9 @@ public SnapWorldDownloadState(
     this.blockchain = blockchain;
     this.snapSyncState = snapSyncState;
     this.metricsManager = metricsManager;
-    this.blockObserverId = OptionalLong.empty();
+    this.blockObserverId = blockchain.observeBlockAdded(createBlockchainObserver());
```

Comment:
nit - can the comment below go before this line

Reply:
or delete - it's not really adding anything, the line of code says the same thing

Reply:
Done. Removed it. Don't it was adding anything